packages/lib/src/toolbar/Toolbar.tsx

```diff
@@ -65,10 +63,8 @@ function Toolbar(props: PropsWithChildren<Props>) {
       </div>
 
       <OverflowMenu>
-        {[
-          ...initialOverflows,
-          ...outOfView.filter((child) => child.type !== Separator),
-        ]}
+        {overflowChildren}
+        {outOfView.filter((child) => child.type !== Separator)}
```

Comment:
Much cleaner: taking advantage of the fact that arrays can be rendered as is in the JSX.